utils/analyze_output_utils.go

```diff
@@ -83,44 +90,63 @@ func (r SingleVersionPackageAnalyzeResult) OutputText(diffType string) error {
 	return TemplateOutput(strResult, "SingleVersionPackageAnalyze")
 }
 
-type StrPackageInfo struct {
+type PackageOutput struct {
+	Name    string
+	Path    string `json:",omitempty"`
```

Comment:
yes, because prior to the comma should go the field name you want displayed, but if if you don't specify anything, it defaults to the struct's field name, and I'm just keeping it the same here